Source/KSCrash/Source/KSCrash/Recording/Tools/BSG_KSMachHeaders.m

```diff
@@ -0,0 +1,182 @@
+//
+//  BSG_KSMachHeaders.m
+//  Bugsnag
+//
+//  Created by Robin Macharg on 04/05/2020.
+//  Copyright © 2020 Bugsnag. All rights reserved.
+//
+
+#import <mach-o/dyld.h>
+#import <dlfcn.h>
+#import <Foundation/Foundation.h>
+#import "BSG_KSMachHeaders.h"
+
+void bsg_initialize_binary_images_array(BSG_Mach_Binary_Images *array, size_t initialSize) {
```

Comment:
Do we need to pass the array into these functions? Can they not access the static directly?

Reply:
Good spot.  Initially written to be a more general-purpose container, the storage is file-static so yes, parameter removed.